README.md

```diff
@@ -1,71 +1,78 @@
 
-- freedom to read, write, and use the format (NO PATENT)
-- freedom to read, understand, and share the format's specification (NO PAYWALL)
-- freedom to modify and distribute the format
+- NO PATENT -- freedom to read, write, and use the format
+- NO PAYWALL -- freedom to read, understand, and share the format's specification
+- NO PROHIBITIONS -- freedom to modify and distribute the format
 
 We propose accelerating the implementation of lossless audiovisual formats with the following coordinated efforts:
 
-- coordinated registration efforts for lossless formats with the principle audiovisual standards organizations (at least IANA, IETF, ISO, SMPTE)
+- coordinated registration efforts for lossless formats with the principle audiovisual standards organizations (IANA, IETF, ISO, SMPTE, et al)
 - open documentation efforts of open, lossless audiovisual formats for relevant communities
 - advocacy and promotion of open, lossless audiovisual formats to media application developers (both proprietary and open source)
 
-Following with the work of the PREFORMA Project we are focused on 3 open lossless formats:
+Following with the work of the [PREFORMA project](http://preforma-project.eu/) we are focused on 3 open lossless formats:
 
-- Matroska, a file container able to transport different media stream
+- Matroska, a file container with the ability to transport various media streams
 - FFV1, a lossless video coding format
 - FLAC, a lossless audio coding format
 
-This focus is not specific to the arrangement of these formats but regards their use in other contents as well, such as FFV1 in MXF or FLAC in QuickTime.
+This focus is not specific to the arrangement of these formats but regards their use in other contents as well, such as using FFV1 in the MXF format or FLAC wrapped in the QuickTime format.
+
+This page lists a summary of what we are doing regarding the registration of these formats in standard bodies used by people and organizations working with open lossless formats.
 
-This page lists a summary about what we are doing about the registering of these formats in standard bodies used by open lossless formats people.  
-Note: standardization of formats themselves is excluded of this summary (it is part of the [PREFORMA project](http://preforma-project.eu/)).
+*Note: Standardization of the formats themselves is excluded from this summary. It is part of the [PREFORMA project](http://preforma-project.eu/)).*
 
 ## Proposed Actions
 
 ### Matroska
 
-- FFV1: currently using the AVI compatibility layer (similar to early version of AVC support), discussion to have a dedicated codec identifier for avoiding it (similar to current version of AVC support).  
+- FFV1: Currently using the AVI compatibility layer (similar to early version of AVC support), with active discussion about instead having a dedicated codec identifier (also similar to current version of AVC support).  
```

Comment:
Ha, this is already out of date since initially written by https://github.com/Matroska-Org/matroska-specification/commit/51a600106bfa66f58a7f2c8e05fab091ab059ebd. I can update in a subsequent PR.